clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/artifact/ArtifactReplacerFactory.java

```diff
@@ -40,57 +40,57 @@
 
   public static Replacer dockerImageReplacer() {
     return Replacer.builder()
-        .replacePath("$.spec.template.spec.containers.[?( @.image == \"{%name%}\" )].image")
-        .findPath("$.spec.template.spec.containers.*.image")
+        .replacePath("$(.spec.jobTemplate)?.spec.template.spec.containers.[?( @.image == \"{%name%}\" )].image")
```

Comment:
This seems to have broken the artifact replacement tests: https://travis-ci.org/spinnaker/clouddriver/builds/388392075#L1724